### ISSUE TITLE
test: test-mount-utils fails in assert(mnt_id == mnt_id2)

### ISSUE THREAD

**Issue by yuwata** [systemd/systemd#7431]
### Submission type

 - Bug report

### systemd version the issue has been seen with

After #7395 is merged.

### Used distribution

Fedora 27 x86_64

### In case of bug report: Expected behaviour you didn't see

test-mount-util passes.

### In case of bug report: Unexpected behaviour you saw

```
The output from the failed tests:
 79/250 test-mount-util                         FAIL    15.45 s
--- command ---
SYSTEMD_KBD_MODEL_MAP='/builddir/build/BUILD/systemd-abe4a74f801729243b5f67247f980a0d2462e74e/src/locale/kbd-model-map' SYSTEMD_LANGUAGE_FALLBACK_MAP='/builddir/build/BUILD/systemd-abe4a74f801729243b5f67247f980a0d2462e74e/src/locale/language-fallback-map' PATH='/builddir/build/BUILD/systemd-abe4a74f801729243b5f67247f980a0d2462e74e/x86_64-redhat-linux-gnu:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/builddir/.local/bin:/builddir/bin' /builddir/build/BUILD/systemd-abe4a74f801729243b5f67247f980a0d2462e74e/x86_64-redhat-linux-gnu/test-mount-util
--- stderr ---
mnt id of /sys/dev is 1130
mnt id of /sys/fs/cgroup/perf_event is 949
mnt id of /sys/devices is 1131
mnt id of /sys/fs/cgroup/cpu,cpuacct is 951
mnt id of /sys/fs/cgroup/hugetlb is 952
mnt id of /proc/sysrq-trigger is 1125
mnt id of /dev/console is 1116
mnt id of /sys/fs/cgroup/blkio is 953
mnt id of /sys/fs/cgroup/devices is 945
mnt id of /sys/fs/cgroup/memory is 947
mnt id of / is 1110
mnt id of /sys/fs/cgroup/net_cls,net_prio is 948
mnt id of /sys/fs/cgroup/freezer is 946
mnt id of /proc/sys/kernel/random/boot_id is 956
Assertion 'mnt_id == mnt_id2' failed at ../src/test/test-mount-util.c:100, function test_mnt_id(). Aborting.
-------
```

**Comment by poettering** [comment]
i am on it

**Comment by poettering** [comment]
Fix waiting in #7436. PTAL
